mvp-spec/partials-updater/src/index.ts

```diff
@@ -32,6 +33,7 @@ async function update() {
 (async () => {
     try {
         await update();
+        await checkUnusedPartials()
```

Comment:
Would be much simpler to just add the used partials in the `update()` method, as soon as they are used, don't you think?
I guess you wanted to keep the two methods with a single responsibility, but I think it's to the price of more complex code that will go through the whole parsing of documents **twice** instead of once.

I would recommend keeping a global variable that holds used partials and update it as soon as one is used in a document.

Reply:
Simpler to do but harder to maintain and add new features. 

Few points:
* We don't have unit tests on this script. Therefore it is more dangerous to mix responsibilities for the sake of saving a few lines. 
* We don't care if the script parses the docs once or twice: the script runs on our desktops in one second from time to time.
* What is important is to have atomical operations that can be easily combined to create quickly new features quickly. Mixing responsibilities would slow us down.

BTW a few more lines don't mean more complexity. :-)

Reply:
Fine for me!